pkg/apis/garden/types.go

```diff
@@ -1259,6 +1259,24 @@ type CloudControllerManagerConfig struct {
 // KubeControllerManagerConfig contains configuration settings for the kube-controller-manager.
 type KubeControllerManagerConfig struct {
 	KubernetesConfig
+	// HorizontalPodAutoscalerConfig contains horizontal pod autoscaler configuration settings for the kube-controller-manager.
+	// +optional
+	HorizontalPodAutoscalerConfig *HorizontalPodAutoscalerConfig
+}
+
+// HorizontalPodAutoscalerConfig contains horizontal pod autoscaler configuration settings for the kube-controller-manager.
+// Note: Descriptions were taken from the Kubernetes documentation.
+type HorizontalPodAutoscalerConfig struct {
+	// The period since last downscale, before another downscale can be performed in horizontal pod autoscaler.
+	DownscaleDelay time.Duration `json:"downscaleDelay"`
```

Comment:
It's getting called "downscale stabilization" in 1.12, hence, should we immediate call it `downscaleStabilization` as well?

Reply:
Downscale stabilisation means some subtly different from a simple downscale delay.

I think we should not use the same field for both. I would suggest that we should keep `downscaleDelay` for 1.11 and later introduce another field `downscaleStabilization` in 1.12. It would be confusing otherwise. 

WDYT?

We should validate that the flags match kubernetes version, of course.

Reply:
OK, agreed.